src/backend/DocDownloaderRedisCache/src/main/java/ca/bc/gov/ag/courts/controller/JobController.java

```diff
@@ -1,84 +1,86 @@
-import org.springframework.web.bind.annotation.PutMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
-import ca.bc.gov.ag.courts.model.Job;
-import ca.bc.gov.ag.courts.repo.JobRepository;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
 
 @RestController
 public class JobController {
 
-	Logger logger = LoggerFactory.getLogger(JobController.class);
-
-	@Autowired
-	JobRepository repo;
-
-	@GetMapping("/jobs")
-	public ResponseEntity<List<Job>> getJobs() {
-		
-		logger.info("get Jobs called"); 
-		
-		List<Job> result = 
-				  StreamSupport.stream(repo.findAll().spliterator(), false)
-				    .collect(Collectors.toList());
-		
-		return new ResponseEntity<List<Job>>(result, HttpStatus.OK);
-		
-	}
-	
-	@GetMapping(path="/jobs/{id}", produces = "application/json")
-	public ResponseEntity<Optional<Job>> getJobById(@PathVariable String id) {
-		
-		logger.info("get Job for id " + id + " called"); 
-		
-		return new ResponseEntity<Optional<Job>>(repo.findById(id), HttpStatus.OK);
-	}
-	
-	@PostMapping(path="/job", consumes = "application/json", produces = "application/json")
-	public String postJob(@RequestBody Job job) {
-
-		logger.info("Add Job called. Job being added to the repo: " + job.toString() );
-
-		repo.findById(job.getId()).orElseThrow(() -> new IllegalArgumentException("Job already exists for id: " + job.getId()));
-		repo.save(job);
-		
-		return "New job added";
-	}
-	
-	@PutMapping(path="/job", consumes = "application/json", produces = "application/json")
-	public String updateJob(@RequestBody Job job) {
-
-		logger.info("Update Job called. Job being updated is : " + job.toString() );
-		repo.findById(job.getId()).orElseThrow(() -> new IllegalArgumentException("Job not found for id: " + job.getId()));
-		repo.save(job); // note - use of same repo method as add. 
-		
-		return "Job updated";
-	}
-	
-	@DeleteMapping(path="/job/{id}", produces = "application/json")
-	public String deleteJob(@PathVariable String id) {
-
-		logger.info("Delete Job called. Job being deleted is : " + id ); 
-		repo.findById(id).orElseThrow(() -> new IllegalArgumentException("Job not found for id: " + id));
-		repo.deleteById(id); // note - use of same repo method as add.
-		
-		return "Job deleted";
-	}
+    Logger logger = LoggerFactory.getLogger(JobController.class);
+
+    @Autowired
+    JobRepository repo;
+
+    @GetMapping("/jobs")
+    public ResponseEntity<List<Job>> getJobs() {
+
+        logger.info("get Jobs called");
+
+        List<Job> result =
+                StreamSupport.stream(repo.findAll().spliterator(), false)
+                        .collect(Collectors.toList());
+
+        return new ResponseEntity<List<Job>>(result, HttpStatus.OK);
+
+    }
+
+    @GetMapping(path = "/jobs/{id}", produces = "application/json")
```

Comment:
I'd recommend to change this to a [MediaType](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/MediaType.html), like - `MediaType.APPLICATION_JSON`. [JavaDoc](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/MediaType.html#APPLICATION_JSON)